src/GraphMetadataViewExtension/GraphMetadataView.xaml

```diff
@@ -72,68 +124,100 @@
                     Grid.Row="0">
             <GroupBox Header="Description"
                       Height="100">
-                <TextBox Text="{Binding GraphDescription, Mode=TwoWay, FallbackValue='Graph Description'}"/>
+                <TextBox Text="{Binding GraphDescription, Mode=TwoWay, FallbackValue='Graph Description'}" Style="{StaticResource TextBoxStyle}"/>
             </GroupBox>
             <GroupBox Header="Image">
                 <controls:ImageSelector MaxHeight="150"
                                         MinHeight="100"
                                         Image="{Binding Thumbnail, Mode=TwoWay}"/>
             </GroupBox>
             <GroupBox Header="Author Name">
-                <TextBox Text="{Binding GraphAuthor, Mode=TwoWay}" />
+                <TextBox Text="{Binding GraphAuthor, Mode=TwoWay}" Style="{StaticResource TextBoxStyle}" />
             </GroupBox>
-            <GroupBox Header="Learn more URL">
-                <TextBox Text="{Binding HelpLink, Converter={StaticResource UriToStringConverter}, Mode=TwoWay}" />
-            </GroupBox>
-            <Separator />
+            <StackPanel>
```

Comment:
We dont need this stackPanel, that was just something i used in the example i showed, instead we should move the `<ControlTemplate x:Key="errorTemplate">` up to `ResourceDictionary` and reference it from there